contracts/EscrowAndStaking/StakingThales.sol

```diff
@@ -582,18 +590,78 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
             iEscrowThales.totalEscrowBalanceNotIncludedInStaking()
         );
 
-        //Actions taken on every closed period
         currentPeriodRewards = fixedPeriodReward;
         _totalUnclaimedRewards = _totalUnclaimedRewards.add(currentPeriodRewards.add(periodExtraReward));
-
         currentPeriodFees = feeToken.balanceOf(address(this));
-
         totalStakedLastPeriodEnd = _totalStakedAmount;
         totalEscrowedLastPeriodEnd = _totalEscrowedAmount;
```

Comment:
modified and `totalEscrowedLastPeriodEnd` is directly assigned instead of using _totalEscrowedAmount 
```
totalEscrowedLastPeriodEnd = iEscrowThales.totalEscrowedRewards().sub(
            iEscrowThales.totalEscrowBalanceNotIncludedInStaking()
        );
        ```